app/src/editor/canvas/components/Cables.jsx

```diff
@@ -30,6 +30,10 @@ export function Cable({ cable }) {
                 to.left,
                 to.top,
             )}
+            stroke="#525252"
+            fill="none"
+            strokeWidth="1"
```

Comment:
Why move from css?

Reply:
Apparently attributes have lower prio than styling, so in order to customize strokes or fills I'd have to define another class.

Reply:
👍

Reply:
you could maybe apply the customisation as an inline style property